src/java.base/share/classes/com/sun/crypto/provider/RC2Parameters.java

```diff
@@ -221,8 +221,7 @@ protected String engineToString() {
                 + encoder.encodeBuffer(iv) + "]");
 
         if (version != 0) {
-            sb.append(LINE_SEP + "version:" + LINE_SEP
-                + version + LINE_SEP);
+            sb.append(LINE_SEP).append("version:").append(LINE_SEP).append(version).append(LINE_SEP);
```

Comment:
Well, the original code is easier to read. Probably no difference performance-wise given it's just  a few known strings. Why bother especially when the StringBuilder constructor also uses `+`?

Reply:
The original code really is easier to read. I'll revert the change.